docroot/themes/humsci/humsci_basic/src/scss/elements/_local-tasks.scss

```diff
@@ -0,0 +1,53 @@
+// Drupal brand colors
+// https://www.drupal.org/drupalorg/style-guide/colors
+$hb-color--drupal-light: #53B0EB;
+$hb-color--drupal-dark: #0678BE;
+
```

Comment:
can you write one tiny comment here about what the local-tasks are? I wish the naming were different 😂